.gitlab-ci.yml

```diff
@@ -6,10 +6,10 @@ stages:
 
 .terraform:
   before_script:
-    - schutzbot/ci_details.sh > ci-details-before-run
+    - ci/ci/schutzbot/ci_details.sh > ci-details-before-run
```

Comment:
is this correct? should it not be only one ci/?

Reply:
That was not correct :)